recipes/hidefix/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "hidefix" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hidefix-{{ version }}.tar.gz
+  sha256: a9162d5f881e4b9890d8852f543b85e77ab565ae6b52bfe7420e64002fa3a5ed
+
+build:
+  number: 0
+  # I think all these files are standard windows files, and not really provided by a package. They are probably linked in
+  # by some package providing random numbers. Since they are standard windows files, they should be safe to ignore?
+  missing_dso_whitelist:
```

Comment:
Adding the c compiler on Windows should fix this.

Reply:
Thanks. Trying.

Reply:
Seems to have done the trick.